src/core/thread/mle.cpp

```diff
@@ -200,6 +173,17 @@ Mle::Mle(Instance &aInstance)
     // to the Link- and Realm-Local All Thread Nodes multicast addresses.
 }
 
+void Mle::InitNetifUnicastAddress(Ip6::NetifUnicastAddress &aAddress)
```

Comment:
Should we also leverage this for other components? For example:
https://github.com/openthread/openthread/blob/7dc3b2696ade4a524b8e2e33cd73bed2668af408/src/core/meshcop/border_agent.cpp#L355-L361

Reply:
Thinking more on this, I think may be best to add one or more such `Init()` methods in `NetifUnicastAddress` class itself. This way can be used by all other classes.